curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/complete-a-promise-with-resolve-and-reject.english.md

```diff
@@ -32,9 +32,9 @@ Make the promise handle success and failure.  If <code>responseFromServer</code>
 
 ```yml
 tests:
-  - text: <code>resolve</code> should be executed when the <code>if</code> condition is <code>true</code>.
+  - text: <code>resolve</code> should be executed with matching string as message when the <code>if</code> condition is <code>true</code>.
```

Comment:
```suggestion
  - text: <code>resolve</code> should be called with the expected string when the <code>if</code> condition is <code>true</code>.
```
Nitpicky, I know, but I wasn't sure what matching or message meant in this context.

Reply:
I concur!